xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.h

```diff
@@ -144,7 +156,8 @@ class CYUV2RGBShader : public CWinShader
   CPoint              m_dest[4];
   EBufferFormat       m_format;
   float               m_texSteps[2];
-  COutputShader *m_pOutShader;
+  COutputShader      *m_pOutShader;
+  bool                m_isNearest;
```

Comment:
Hm, just felt that inconsistency is less preferred than strictly following code guidelines on this one.
Will revert it regardless.